markets/perps-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -72,36 +72,36 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
 
     function setLiquidationParameters(
         uint128 marketId,
-        uint256 initialMarginFraction,
-        uint256 maintenanceMarginFraction,
+        uint256 initialMarginRatioD18,
+        uint256 maintenanceMarginRatioD18,
         uint256 liquidationRewardRatioD18,
         uint256 maxLiquidationLimitAccumulationMultiplier,
         uint256 maxSecondsInLiquidationWindow
     ) external override {
         PerpsMarket.load(marketId).onlyMarketOwner();
         PerpsMarketConfiguration.Data storage config = PerpsMarketConfiguration.load(marketId);
 
-        config.initialMarginFraction = initialMarginFraction;
-        config.maintenanceMarginFraction = maintenanceMarginFraction;
+        config.initialMarginRatioD18 = initialMarginRatioD18;
+        config.maintenanceMarginRatioD18 = maintenanceMarginRatioD18;
         config.liquidationRewardRatioD18 = liquidationRewardRatioD18;
         config
             .maxLiquidationLimitAccumulationMultiplier = maxLiquidationLimitAccumulationMultiplier;
         config.maxSecondsInLiquidationWindow = maxSecondsInLiquidationWindow;
         emit LiquidationParametersSet(
             marketId,
-            initialMarginFraction,
-            maintenanceMarginFraction,
+            initialMarginRatioD18,
+            maintenanceMarginRatioD18,
             liquidationRewardRatioD18,
             maxLiquidationLimitAccumulationMultiplier,
             maxSecondsInLiquidationWindow
         );
     }
 
-    function setLockedOiPercent(uint128 marketId, uint256 lockedOiPercent) external override {
+    function setLockedOiRatioD18(uint128 marketId, uint256 lockedOiRatioD18) external override {
```

Comment:
no need to change name of setter.  it can just stay `setLockedOiRatio`.  the `D18` append is helpful mostly for devs looking at the code; and for auditors

Reply:
also helpful in interfaces where external integrators can see what value type it is